pyServer/manifests/windows/windowsupdate

```diff
@@ -81,7 +81,7 @@ copy,/Windows/System32/Sysprep/Panther/IE/setuperr.log
 copy,/Windows/System32/Sysprep/Sysprep_succeeded.tag
 
 echo,### Plug and Play ###
-copy,/Windows/INF/netcfg*.*etl
+copy,/Windows/INF/netcfg*.*etl,noscan
```

Comment:
Since there is some pattern to the types of files we are skipping for credscan, maybe we can come up with a way to dynamically determine if a file should be added to the credscan file list or not based on file extension? (i.e. *.etl, *edb, and registry hives) This way we don't put the burden on future manifest contributors (primarily CSS advisors) to determine if a file should be cred scanned or not? Just food for thought :)

Reply:
I thought about this option. I mentioned it in one of scrum writeup in email ( that day I happened not able to attend in person). I figured that there are all kind of files out there in different OS VMs, different Linux distros, files created by different apps/agents. What I have seen so far, the file extensions are not 100% indictive of file content. Secondly, if later we need to update/add file extension to the exclusion, to be thorough, we need to test all files/manifests to see if we exclude too many/too encompassing, or when we need to change file extensions or remove file extension, same type of regression tests need to happen. CSS engineers may not know the scope that needs to be tested. This could put burden on them, although the intent was to reduce the burden on them/help them when they later create manifests. Also we may expand this functionality to exclude beyond binary files like files that don't contain secrets for sure. with that the chances are higher that different extensions step on each other. What we have now ( marking each file explicitly) is very explicit, and don't have to worry about regression on other files. that was my thought process.